src/test/rustdoc-ui/intra-doc/macro-rules.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+#![deny(rustdoc::broken_intra_doc_links)]
+
+#[macro_export]
```

Comment:
apparently this `macro_export` isn't necessary for the test to fail :confused: it's unclear to me why *any* macros at all work if something this simple fails.

Reply:
ok, this appears to be an actual mcve:
```rust
#![allow(rustdoc::private_intra_doc_links)]
macro_rules! foo {
    () => {};
}


/// [foo!]
pub fn baz() {}
```